### PR TITLE
feat: copy minified file to example

### LINKED ISSUE #176: Error while running the browser example
On the browser console I see this:
Firefox cannot establish a connection to the server at ws://127.0.0.1:8854/

On the terminal, this is what I see:
"GET /axis-cgi/rtspwssession.cgi?1561702741825" Error (404): "Not found"

Please suggest what am I doing wrong.

### LINKED ISSUE #391: Unable to run examples
Hi there, I'm trying to follow the instructions to host the examples on my machine. I am trying to follow the instructions exactly. 
1. yarn add media-stream-library. This works, it adds the project.
2. yarn examples (from top folder). Examples not found. Okay no big deal, so I navigate to example folder
3. yarn examples (from /node_modules/media-stream-library). Error, webpack: not found. 
4. yarn add webpack --dev (from same folder as above). Seems to work. 
5. yarn examples. I get a ton of compile errors. The first few will be pasted below. 

What am I doing wrong here? I have an RTSP feed ready to play and I want to test it but can't get the examples up and running. Any help is appreciated, thank you.

ERROR in ./lib/utils/protocols/sdp.ts 4:0
Module parse failed: The keyword 'interface' is reserved (4:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import { NtpSeconds, seconds } from './ntp'
| 
> interface ConnectionField {
|   // c=<nettype> <addrtype> <connection-address>
|   networkType: 'IN'
 @ ./lib/utils/protocols/index.ts 1:0-21 1:0-21
 @ ./lib/utils/index.browser.ts
 @ ./lib/index.browser.ts

ERROR in ./lib/components/canvas/index.ts 8:0
Module parse failed: The keyword 'interface' is reserved (8:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import { VideoMedia } from '../../utils/protocols/sdp'
| 
> interface BlobMessage {
|   readonly blob: Blob
|   readonly ntpTimestamp: number
 @ ./lib/components/index.browser.ts 5:0-24 5:0-24
 @ ./lib/index.browser.ts

ERROR in ./lib/components/h264depay/index.ts 10:23
Module parse failed: Unexpected token (10:23)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export class H264Depay extends Tube {
|   constructor() {
>     let h264PayloadType: number
|     let idrFound = false
| 
 @ ./lib/components/index.browser.ts 6:0-27 6:0-27
 @ ./lib/index.browser.ts

ERROR in ./lib/components/jpegdepay/index.ts 10:23
Module parse failed: Unexpected token (10:23)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export class JPEGDepay extends Tube {
|   constructor() {
>     let jpegPayloadType: number
|     let packets: Buffer[] = []
|     let jpegDepay: (
 @ ./lib/components/index.browser.ts 8:0-27 8:0-27
 @ ./lib/index.browser.ts

ERROR in ./lib/pipelines/rtsp-mjpeg-pipeline.ts 11:24
Module parse failed: Unexpected token (11:24)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export class RtspMjpegPipeline extends RtspPipeline {
>   constructor(rtspConfig?: RtspConfig) {
|     super(rtspConfig)
| 
 @ ./lib/pipelines/index.browser.ts 5:0-37 5:0-37
 @ ./lib/index.browser.ts

ERROR in ./lib/components/basicdepay/index.ts 12:28
Module parse failed: Unexpected token (12:28)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| export class BasicDepay extends Tube {
>   constructor(rtpPayloadType: number) {
|     if (rtpPayloadType === undefined) {
|       throw new Error('you must supply a payload type to BasicDepayComponent')
 @ ./lib/components/index.browser.ts 4:0-28 4:0-28
 @ ./lib/index.browser.ts

### LINKED ISSUE #412: Trying to run examples/browser/camera/simple.html
Hello @steabert

i want test this file and my step was 

1- download https://github.com/AxisCommunications/media-stream-library-js
2- cd media-stream-library-js
3- run npm install     to install all package
4- yarn examples       it works and show this message  
yarn run v1.22.0
$ http-server examples/browser
Starting up http-server, serving examples/browser
Available on:
  http://192.168.1.118:8080
  http://192.168.52.1:8080
  http://192.168.199.1:8080
  http://192.168.1.106:8080
  http://127.0.0.1:8080

5- after that i go to this address 
http://localhost:8080/camera/simple.html

and enter my camera ip address
click play button 

but nothing happened  i check console in chrome
and say  
1-  Uncaught SyntaxError: Unexpected token '.    media-stream-library.min.js:1' 

2- Uncaught TypeError: Cannot destructure property 'pipelines' of 'window.mediaStreamLibrary' as it is undefined.        http://simple-player.js:1/

what is problem ????

!!! i can see my  ip  camera  in vlc  with this address and it is ok  
rtsp://admin:test@192.168.1.64:554/cam/realmonitor?channel=1&subtype=0
.

### PR DESCRIPTION
**Pull request by lekoaf**
Instead of using a symlink, which doesn't work on Windows, we copy the minified file to the examples folder.

Fixes #412
Fixes #391
Fixes #176

### CODE REVIEW

**Comment by steabert** [comment]
It seems the copy plugin is run before the output is created. Maybe just run a `cp` after webpack was run in the script?

**Comment by lekoaf** [comment]
> It seems the copy plugin is run before the output is created. Maybe just run a `cp` after webpack was run in the script?

That's strange. It works locally. Oh, maybe it worked locally because I did `yarn build` before. :see_no_evil: 

Yupp, I'll see if I can do a bash script instead.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "lint": "yarn eslint && yarn prettier:check",
     "test": "jest --coverage",
     "verify": "yarn lint && yarn test",
-    "build": "yarn build:esm && yarn build:cjs && yarn build:es5 && yarn build:bundle",
+    "build": "yarn build:esm && yarn build:cjs && yarn build:es5 && yarn build:bundle && sbin/copyMinified.sh",
```
Comment (steabert) [inline package.json:25]: probably better to move the copying to the `build:bundle` (it only affects the bundle)

Comment (lekoaf) [reply package.json]: Done.

File: sbin/copyMinified.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (rikteg) [inline sbin/copyMinified.sh:1]: Is this script portable so it will work on windows?

Comment (lekoaf) [reply sbin/copyMinified.sh:1]: Yes.

Comment (rikteg) [reply sbin/copyMinified.sh:1]: ok

**Review by steabert — APPROVED** [review]

**Pull request merged**